### PR TITLE
Multiple sql supports

### PR DESCRIPTION
**Pull request by huangxinV587**

### CODE REVIEW

**Comment by huangxinV587** [comment]
在notebook和shell的效果演示
image
image

**Comment by tonyyang-svail** [comment]
@huangxinV587 Please do one fix at a time. Please also add the corresponding issue explaining the significance of this PR.

File: sqlflow/client.py [hunk 1]
```diff
@@ -92,31 +94,46 @@ def __init__(self, server_url=None):
...
         >>> client.execute("select * from iris limit 1")
+        >>> client.execute("select * from iris limit 1; select count(*) from iris")
 
         """
-        stream_response = self._stub.Run(pb.Request(sql=operation))
-        first = next(stream_response)
-        if first.WhichOneof('response') == 'message':
-            _LOGGER.info(first.message.message)
+        return_rows = []
+        for one_operation in operation.split(";") :
```
Comment (tonyyang-svail) [inline sqlflow/client.py]: what if `;` is wrapped inside the quotation marks?

File: setup.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
     'grpcio >=1.17, <2',
     'ipython>=1.0',
     'prettytable',
+    'sqlparse',
```
Comment (wangkuiyi) [inline setup.py]: 这个sqlparse是一个第三方的package吗？

**Comment by ktong** [comment]
Personally, I do not like to enable client.execute to support multiple sql. The cli already supports pass multiple sqls (https://github.com/wangkuiyi/pysqlflow/blob/develop/sqlflow/__main__.py#L6). The only change need is modify __main__,py to call client.execute for each sql passed from cli.

File: setup.py (hunk 2 above)
Comment (ktong) [reply setup.py]: Does sqlparse support customized training SQL? or Mysql special sql like use database and postgres sql like \dt? If it really need supports multiple sql, I would like to implement it on server side. Just pass anything in cell to server and server runs sqls sequentially.

**Pull request closed without merging**